docs/UserGuide.md

```diff
@@ -221,14 +224,14 @@ _Details coming soon ..._
-| **List**             | `list`                                                                                                                                                                              |
-| **Help**             | `help`                                                                                                                                                                              |
+| Action               | Format, Examples                                                                                                                                                                               |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**              | `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]` <br> e.g., `add n/John Doe p/98765432 e/johnd@example.com rm/ 05-73 dr/19/05/20 - 24/05/22 ng/3`    |
+| **Clear**            | `clear`                                                                                                                                                                                        |
+| **Delete**           | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                            |
+| **Edit**             | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [rm/ROOM] [dr/DATE_RANGE] [ng/NUMBER_OF_GUESTS] [rc/IS_ROOM_CLEAN] [rq/REQUEST]`<br> e.g.,`edit 2 e/jameslee@example.com dr/13/09/22 - 15/09/22 ng/5` |
+| **Bill**             | `bill INDEX b/BILL`<br> e.g., `bill 2 b/99.99`                                                                                                                                                 |
+| **Find**             | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                     |
+| **MarkRoomsUnclean** | `markRoomsUnclean`                                                                                                                                                                             |
+| **List**             | `list`                                                                                                                                                                                         |
+| **Help**             | `help`                                                                                                                                                                                         |
```

Comment:
Should we add exit here as well?

Reply:
Yes, we should, but that part is left for Jerry to edit!